e2e/stack.go

```diff
@@ -243,8 +244,11 @@ func (s *Stack) runMonomer(ctx context.Context, env *environment.Env, genesisTim
 	env.DeferErr("close tx db", txdb.Close)
 	mempooldb := dbm.NewMemDB()
 	env.DeferErr("close mempool db", mempooldb.Close)
-	ethstatedb := rawdb.NewMemoryDatabase()
-	env.DeferErr("close eth state db", ethstatedb.Close)
+	rawDB := rawdb.NewMemoryDatabase()
+	env.DeferErr("close raw db", rawDB.Close)
+	trieDB := triedb.NewDatabase(rawDB, nil)
+	env.DeferErr("close trieDB", trieDB.Close)
+	ethstatedb := state.NewDatabaseWithNodeDB(rawDB, trieDB)
```

Comment:
nit: could you use lowercase `db` instead of `DB` to have parity with the rest of the database names here (and also in `integrations.go`)?